app/src/main/java/com/duckduckgo/app/browser/remotemessage/RemoteMessagingModel.kt

```diff
@@ -28,7 +28,7 @@ import dagger.SingleInstanceIn
 import kotlinx.coroutines.withContext
 import javax.inject.Inject
 
-@SingleInstanceIn(AppScope::class)
+@SingleInstanceIn(FragmentScope::class)
```

Comment:
This could be also `ActivityScope` but I decided to tighten the scoping purposely